src/platform/plugins/shared/unified_search/public/plugin.ts

```diff
@@ -62,7 +62,7 @@ export class UnifiedSearchPublicPlugin
 
   public start(
     core: CoreStart,
-    { data, dataViews, uiActions, screenshotMode }: UnifiedSearchStartDependencies
+    { data, dataViews, uiActions, screenshotMode, cps }: UnifiedSearchStartDependencies
```

Comment:
Nit. Can we assign a more meaningful variable name in this destructuring?

```suggestion
    { data, dataViews, uiActions, screenshotMode, cps: crossProjectSearch }: UnifiedSearchStartDependencies
```